code/script.js

```diff
@@ -26,22 +71,98 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
+  
+// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+ 
+chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Chat starts here
+//Here you can see the questionNumbers and there order set with a number 1-3
+
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1;
+  showMessage(`Hello, what's your name?`, 'bot');
+}
+
+const usrInput = (answer) => {
+  showMessage(answer, 'user')
+}
+
+const question1 = (question) => {
+  const input = document.getElementById('user-input').value
+  questionNumber++
+  showMessage(`Okay, ${input}, what do you want!?`, 'bot')
+}
+  
+const question2 = (question) => {
+  questionNumber++
+  showMessage(`I don't care. Which one of these do you need help with?`, 'bot')
+  form.style.display='none'
+  factButtons.style.display='flex'
 }
 
-// Set up your eventlisteners here
+//Here we set a setTimeout for each answer - in comparison with the bots question, which are all set in the function above. 
```

Comment:
Nice that you have used a different approach to show that you know how to do it in several ways. Is there a specific reason why you use setTimeout here differently? Since you've already created a function above which you could have used on all of the bots answers for example?